cogs/embeds.py

```diff
@@ -367,10 +358,7 @@ async def seize(self, ctx, amount: int, member: discord.Member):
             await ctx.channel.send(
                 "Minumum amount of credits to remove is 1.")
             return
-        dbprof_member = await db.get_member(member.guild.id, member.id)
-        await db.set_member(
-            dbprof_member['guild_id'], dbprof_member['member_id'], 'cash',
-            dbprof_member['cash'] - amount)
+        await db.remove_currency(ctx.guild.id, member.id, amount)
         await ctx.channel.send(
             f"{amount} credit(s) has been taken from {member.mention}.")
```

Comment:
I'd like to see this string rephrased as `Removed {amount} credits from {member.mention}.`
Don't worry about accounting for the grammar of 1 credit being removed, I think the readability for the other infinite amount of credits above 1 is more important.